quiz/docker-compose.yaml

```diff
@@ -1,5 +1,5 @@
 services:
-  quiz_web:
+  web:
```

Comment:
removed quiz prefix, 
Compose by default prefixes all services with the folder name which caused all of the services to be created as 
`quiz-quiz_<name>-1`

Reply:
I find it better, it is more readable in the terminal. Services names were added after @pawelkonior review. It was said then that services name would look better with project names in docker-compose.yaml files

Reply:
`quiz-quiz_web-1`

Reply:
`quiz-web-1`

Reply:
`quiz-web-1` is much better. No argue there 😄